packages/util-dev/src/mockProviders/mockData.ts

```diff
@@ -42,14 +42,16 @@ export const protocolParameters = {
 export const epochRewards = [
   {
     epoch: Cardano.EpochNo(currentEpoch.number - 3),
+    poolId: Cardano.PoolId('pool1h8yl5mkyrfmfls2x9fu9mls3ry6egnw4q6efg34xr37zc243gkf'),
```

Comment:
I see this pool id more times in this PR.
Is this mock the only source of this pool id or is it taken from fixtures (db dump), too?
I'm asking just to be sure this PR is resilient to fixtures re-generation.
Thank you